editor/editor_file_system.cpp

```diff
@@ -1018,6 +1044,10 @@ void EditorFileSystem::_process_file_system(const ScannedDirectory *p_scan_dir,
 
 		EditorFileSystemDirectory::FileInfo *fi = memnew(EditorFileSystemDirectory::FileInfo);
 		fi->file = scan_file;
+		fi->link = da->is_link(path);
```

Comment:
To optimize the scan process, ideally, the link information should be saved in `filesystem_cache8`. Technically, if the last modification time hasn't changed, it's unlikely that the link information would be different, right?

Managing `filesystem_cache8` to ensure backward compatibility while also detecting when new information is missing in the file is not straightforward. In #95678, I used the field at index 7, which already contains some information separated by `<>`. This field can hold multiple pieces of information separated by `<>`.

Please note that I haven't tested the performance impact of checking the link information on disk for each file yet.

Reply:
I did some testing on Window 11 and with this draft, doing the first scan (`_process_file_system`) it only adds a couple seconds for a project of over 60000 files.
Master version: 46sec
This PR: 49sec

Seeing these numbers, I guess that the need for putting the link information into `filesystem_cache8` does not sound a necessity!